packages/eas-cli/graphql-codegen.yml

```diff
@@ -16,7 +14,7 @@ generates:
       dedupeOperationSuffix: true
     hooks:
       afterOneFileWrite:
-        - 'node ./scripts/annotate-graphql-codegen.js'
+        - ./annotate-graphql-codegen.sh
```

Comment:
Confirming this is an intentional change? I see `./scripts/annotate-graphql-codegen.js` in the repo but not `./annotate-graphql-codegen.sh`.

Reply:
nope not intentional, will remove